content/_guides/bots.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Network bots
+category: chanop
+---
+
+We run various IRC bots, some of them have a public facing usage.
+Any requests or questions about our utility bots can be asked in `#libera-bots`.
+
+## ozone
+
+ozone is anti spam bot which kills and klines spammers. ozone is an instance of [Sigyn](https://github.com/Libera-Chat/Sigyn).
+
+When your channel is being targeted by spam bots, you can `/invite ozone`
+**If you run some bots, you should voice them** otherwise ozone could ban them if they are too verbose.
+
+If it bans a legit user, op yourself on your channel `/msg chanserv op $channel` and ask ozone to unkline the  user `/msg ozone unkline $nick`
+
+If you want ozone to stay permanently on your channel, contact us.
+ozone may require a minimum numbers of users on the channel before joining, this varies over time.
+ozone may also enter a channel on it's own if it believes that the channel is under attack.
```

Comment:
> on it's own

please

Reply:
It's a true statement? See https://github.com/Libera-Chat/Sigyn/blob/master/plugin.py#L2575-L2597

Reply:
yes, C&C channels get destroyed that way.

Reply:
...not sure about how true that is but maybe it should be "on its own"

Reply:
Ah yes, `on it's own` is technically saying `on it is own` so really it should be `on its own`. It might've been better if you had led with that though.